vpp/vaapipostprocess_scaler.cpp

```diff
@@ -59,9 +62,29 @@ VAProcColorStandardType fourccToColorStandard(uint32_t fourcc)
     return VAProcColorStandardBT601;
 }
 
+VaapiPostProcessScaler::VaapiPostProcessScaler()
```

Comment:
Put the denosie/de-interlance/... in vaapipostprocess_scaler.cpp/vaapipostprocess_scaler.h maybe is not good idea.